### PR TITLE
mtdev: Add recipe

### PR DESCRIPTION
**Pull request by jwillikers**
Specify library name and version:  **mtdev/1.1.6**

The mtdev package is required to create a libinput package.

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/mtdev/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    def package(self):
+        copy(self, "COPYING", self.source_folder, os.path.join(self.package_folder, "licenses"))
+        autotools = Autotools(self)
+        autotools.install()
+        rm(self, "*.la", os.path.join(self.package_folder, "lib"))
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+
+    def package_info(self):
+        self.cpp_info.libs = ["mtdev"]
+        self.runenv_info.prepend_path("PATH", os.path.join(self.package_folder, "bin"))
```
Comment (RubenRBS) [inline recipes/mtdev/all/conanfile.py:67]: Quick curiosity question, what's being added in bin in this library's case?

**Review by RubenRBS — APPROVED** [review]
Thanks!

File: recipes/mtdev/all/conanfile.py (hunk 1 above)
Comment (jwillikers) [reply recipes/mtdev/all/conanfile.py:67]: A helper utility called `mtdev-test` which can be passed a device path to a multitouch device. I'm not exactly sure what it does when you do it, but I decided to include it in the package. There is no available Meson option to enable / disable including it, at least right now.

Comment (SpaceIm) [inline recipes/mtdev/all/conanfile.py:67]:
```suggestion
```

To remove. Any folder in cpp_info.bindirs is automatically added to PATH env var in VirtualRunEnv or VirtualBuildEnv.

File: recipes/mtdev/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    def validate(self):
+        if self.settings.os not in ["Linux", "FreeBSD"]:
+            raise ConanInvalidConfiguration(f"{self.ref} is not supported on {self.settings.os}.")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        env = VirtualBuildEnv(self)
+        env.generate()
```
Comment (SpaceIm) [inline recipes/mtdev/all/conanfile.py:49]:
```suggestion
```

File: recipes/mtdev/all/conanfile.py (hunk 1 above)
Comment (jwillikers) [reply recipes/mtdev/all/conanfile.py:67]: Thanks @SpaceIm!

**Review by RubenRBS — APPROVED** [review]

**Review by czoido — APPROVED** [review]

**Pull request merged**